static/js/admin.js

```diff
@@ -409,108 +409,119 @@ function getRightSideGradingDiv(element, acid, studentId) {
 
 }
 
-
+function makeOption(text,value,disabledQ){
```

Comment:
These changes seem out of place and very unrelated to the purpose of your PR.  I’m not sure who all of these changes got here.

Reply:
OK, I'll replace this with separate pull requests: is the key fewer and more substantial changes, rather than smaller and more frequent changes? Or do all y'all have like a hundred different branches, one for each fixed typo?